app/components/floor-tooltip/floor-tooltip.tsx

```diff
@@ -39,7 +40,10 @@ const styles = StyleSheet.create({
     fontSize: 20,
     marginBottom: 20,
   },
-})
+  icon: {
+    color: type === "info" ? theme.colors.grey1 : theme.colors.error5,
```

Comment:
I don't think conditional in styles is best practice ( no strong opinion though, but this is my guts feeling)

another way to deal with this would be to have a something like:

```
iconInfo: {
    color: theme.colors.grey1,
  },
iconWarning: {
   colors: theme.colors.error5
}
```